## Motor Params
gain: 1
# negative: more to right postive: more to left
# positive make right wheel do more; negative makes left wheel do more
trim: 0.07 #0.015

## Odom Params
wheel_rad: 0.033
baseline: 0.163
encoder_res: 137

## Movement params